browser/js/common/states/notloggedin/notloggedin.state.js

```diff
@@ -0,0 +1,8 @@
+app.config(function ($stateProvider) {
+	$stateProvider.state('notLoggedIn', {
+		url: '/notloggedin',
+		template: `	<h1>
+						You cannot access this without being logged in!
+					</h1>`
+	})
+})
```

Comment:
this state is purely for handling when the user is not logged in but still tries to access stuff that only logged in users should be able to access. can access using $state.go